data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -93,13 +93,16 @@ then
 
 	rm --recursive --force api-bootstrap
 
+  # we should distinguish between self-managed-high-availability and single-node-developer, but for now they are identical
```

Comment:
@patrickdillon any easy way to distinguish?

Reply:
Apparently there's a very easy way!
https://github.com/openshift/installer/blob/7be2cf71cdef9f5fe0d2d9687615c2bb2741ffcd/data/data/bootstrap/files/usr/local/bin/bootkube.sh.template#L12-L16

Reply:
Awesome!  Thanks.